gson/src/test/java/com/google/gson/functional/ObjectTest.java

```diff
@@ -141,9 +141,9 @@ public void testClassWithTransientFieldsDeserialization() {
 
   @Test
   public void testClassWithTransientFieldsDeserializationTransientFieldsPassedInJsonAreIgnored() {
-    String json = "{\"transientLongValue\":1,\"longValue\":[1]}";
+    String json = "{\"transientLongValue\":5,\"longValue\":[1]}";
```

Comment:
Changed this because otherwise the test is extremely misleading: The point is that the value in JSON is ignored. However, the `ClassWithTransientFields` constructor also initializes the field to 1, so if the JSON contains 1 as well it is not possible to tell where the value came from.